ts/screens/preferences/common.ts

```diff
@@ -20,12 +20,41 @@ const INBOX_CHANNEL = "INBOX";
 const EMAIL_CHANNEL = "EMAIL";
 const PUSH_CHANNEL = "WEBHOOK";
 
+export function getChannelsforServicesList(
+  servicesId: ReadonlyArray<string>,
+  profile: ProfileState
+): BlockedInboxOrChannels {
+  const profileBlockedChannels = pot
```

Comment:
```suggestion
  const profileBlockedChannels = pot.getOrElse(
    pot.mapNullable(
      profile,
      up => (InitializedProfile.is(up) ? up.blocked_inbox_or_channels : {})
    ),
    {}
  );
```
Check if this could be better (without option conversion)

Reply:
yes, it seems more readable too. Updated within https://github.com/teamdigitale/io-app/pull/1269/commits/383772845fa1b1dd59a4d61d5512ba2520b8f0ce